tachyon/zk/plonk/examples/shuffle_circuit.h

```diff
@@ -196,10 +196,10 @@ class ShuffleCircuit : public Circuit<ShuffleCircuitConfig<F, W>> {
 
   std::unique_ptr<Circuit<ShuffleCircuitConfig<F, W>>> WithoutWitness()
       const override {
-    std::vector<std::vector<F>> dummy_original_table = base::CreateVector(
-        W, []() { return base::CreateVector(H, F::Zero()); });
-    std::vector<std::vector<F>> dummy_shuffled_table = base::CreateVector(
-        W, []() { return base::CreateVector(H, F::Zero()); });
+    std::vector<std::vector<F>> dummy_original_table =
+        base::CreateVector(W, []() { return std::vector<F>(H); });
+    std::vector<std::vector<F>> dummy_shuffled_table =
+        base::CreateVector(W, []() { return std::vector<F>(H); });
```

Comment:
I think it can be simplified like the following
```
std::vector<std::vector<F>> dummy_original_table(W, std::vector<F>(H));
```

Reply:
Maybe the separated commits are causing confusion :thinking: oops. I separated cases where the `CreateVector(size, value)` was used and cases where I could collapse a `CreateVector` with a lambda function that was unneeded into two different commits. That is to say, in my third commit I changed this line to what you mentioned.